rerun-failed-run/check-annotations-and-logs/find-errors.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
```

Comment:
for a fail-fast approach may also a `-u` flag would make sense.

Reply:
It makes sense :+1

Reply:
I will add all the options ^^, thx!

Reply:
Fixed.